tests/vary.mjs

```diff
@@ -80,6 +80,72 @@ export default {
         }
       ]
     },
+    {
+      name: 'An optimal HTTP cache stores and correctly serves multiple object variants when the `Vary` response header value depends on one or more request header values',
+      id: 'vary-dynamic',
+      kind: 'optional',
+      browser_skip: true,
+      requests: [
+        {
+          request_headers: [
+            ['v1', 'one'],
+            ['v2', 'two'],
+            ['ctrl', 'a']
+          ],
```

Comment:
Requests that aren't actually performing the test should have `setup: true` so that they don't get shown as false positives/negatives.